setup.py

```diff
@@ -65,6 +65,7 @@
     "pytest>=5.3.2",
     "coverage>=4.5.4",
     "hdf5storage>=0.1.18",
+    "onnxruntime>=1.11.0",
```

Comment:
Since we removed the onnxruntime dep from the requirements, we may want to remove from the setup file as well